docs/local_assembly.tex

```diff
@@ -0,0 +1,77 @@
+\documentclass[nofootinbib,amssymb,amsmath]{revtex4}
+\usepackage{mathtools}
+\usepackage{amsthm}
+\usepackage{algorithm}
+\usepackage{algpseudocode}
+\usepackage{lmodern}
+\usepackage{graphicx}
+\usepackage{color}
+
+%Put an averaged random variable between brackets
+\newcommand{\ave}[1]{\left\langle #1 \right\rangle}
+\newcommand{\HC}{\texttt{HaplotypeCaller}}
+\newcommand{\Mutect}{\texttt{Mutect}}
+\newcommand{\code}[1]{\texttt{#1}}
+
+\newtheorem{lemma}{Lemma}
+\newtheorem{corollary}{Corollary}
+
+\def\SL#1{{\color [rgb]{0,0,0.8} [SL: #1]}}
+\def\DB#1{{\color [rgb]{0,0.8,0} [DB: #1]}}
+
+\begin{document}
+
+\title{Local Assembly in HaplotypeCaller and Mutect}
+\author{David Benjamin\footnote{The author took no part in development of the methods described below -- credit belongs to several others on the GATK team. }}
+\email{davidben@broadinstitute.org}
+\affiliation{Broad Institute, 75 Ames Street, Cambridge, MA 02142}
+
+\date{\today}
+
+\begin{abstract}
+The GATK tool \HC~ and \Mutect~ assemble reads aligned within a window of several hundred base pairs into a de Bruijn graph of local variation.  Local haplotypes correspond to paths in this de Bruijn graph, and the downstream likelihood calculations in both tools involve aligning reads to these haplotypes.  Here we describe how this de Bruijn graph is generated.
```

Comment:
I'm comfortable calling it a de Bruijn graph even if it doesn't follow the wikipedia definition as long as it follows the usage in Pevzner and basically every paper on assembly since then.  Does our graph violate *their* definition?